packages/forms/resources/views/components/checkbox-list.blade.php

```diff
@@ -45,6 +45,10 @@
                     this.updateVisibleCheckboxListOptions()
                     this.checkIfAllCheckboxesAreChecked()
                 })
+
+                $nextTick(() => {
+                    this.checkIfAllCheckboxesAreChecked()
```

Comment:
This is also called at the top of the `init` method. Should that call be removed?